how-to/software/automatic-updates.md

```diff
@@ -341,6 +343,20 @@ The `unattended-upgrades` feature is helpful, does its job, and even sends out r
 An example of such a Fleet Management software for Ubuntu systems is [Landscape](https://ubuntu.com/landscape).
 
 
+(unattended_upgrades_postpone)=
+## Postponable updates
+By default, system updates are applied automatically in the background without any user interaction.
+Starting with Ubuntu 25.04, this behaviour can be changed to prompt active users before starting the update, allowing them to postpone the updates to a later time, up to a limited number of days as configured by the system administrator.
+
+To enable the feature, set the `Unattended-Upgrade::Postpone-For-Days` configuration in `/etc/apt/apt.conf.d/50unattended-upgrades` to a positive number, signifying the number of days that a user of this system is allowed to postpone the automatic updates for. Every time `unattended-upgrades` is run, after determining whether some updates are available to be applied it will prompt the user and shortly wait for an answer. If the user requests to postpone an update, the date and time of this action is recorded so that on future runs `unattended-upgrades` will only prompt the user again if the number of days elapsed from the first postpone request is lower than what is configured in `Unattended-Upgrade::Postpone-For-Days`.
```

Comment:
```suggestion
To enable the feature, set the `Unattended-Upgrade::Postpone-For-Days` configuration in `/etc/apt/apt.conf.d/50unattended-upgrades` to a positive number, signifying the number of days that a user of this system is allowed to postpone the automatic updates for. Every time `unattended-upgrades` is run, after determining whether some updates are available to be applied it will prompt the user and shortly wait for an answer. If the user requests to postpone an update, the date and time of this action is recorded. Then, on future runs, `unattended-upgrades` will only prompt the user again if the number of days since the first postpone request is lower than what is configured in `Unattended-Upgrade::Postpone-For-Days`.
```